Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java

```diff
@@ -24,17 +30,33 @@
  * @author Julia Damerow
  * 
  */
+@PropertySource(value = "classpath:/user.properties")
 @Service
 public class UsersManager implements IUserManager {
 
 	@Autowired
 	private UserDatabaseClient client;
-	private Map<String, String> admins;
-
-	@PostConstruct
-	public void init() {
-		admins = new HashMap<String, String>();
-	}
+	private Map<String, Object> admins;
```

Comment:
@jdamerow  I dont understand this comment. I have created a HashMap for storing the username as key and password as value.